Library/Homebrew/hardware.rb

```diff
@@ -7,28 +7,28 @@ class CPU
     INTEL_32BIT_ARCHS = [:i386].freeze
     INTEL_64BIT_ARCHS = [:x86_64].freeze
     PPC_32BIT_ARCHS   = [:ppc, :ppc32, :ppc7400, :ppc7450, :ppc970].freeze
-    PPC_64BIT_ARCHS   = [:ppc64].freeze
+    PPC_64BIT_ARCHS   = [:ppc64, :ppc64le, :ppc970].freeze
 
     class << self
-      OPTIMIZATION_FLAGS = {
-        native:  "-march=native",
-        nehalem: "-march=nehalem",
-        core2:   "-march=core2",
-        core:    "-march=prescott",
-        armv6:   "-march=armv6",
-        armv8:   "-march=armv8-a",
-      }.freeze
-
       def optimization_flags
-        OPTIMIZATION_FLAGS
+        {
+          native:  arch_flag("native"),
+          nehalem: "-march=nehalem",
+          core2:   "-march=core2",
+          core:    "-march=prescott",
+          armv6:   "-march=armv6",
+          armv8:   "-march=armv8-a",
+          ppc64:   "-mcpu=powerpc64",
+          ppc64le: "-mcpu=powerpc64le",
+        }
       end
```

Comment:
```suggestion
      end
      alias generic_optimization_flags optimization_flags
```

then you could reuse it in the Linux code above and modify the value